doc/nrf/ug_matter_configuring_protocol.rst

```diff
@@ -14,11 +14,12 @@ This page describes what is needed to start working with Matter in the |NCS|.
 Mandatory configuration
 ***********************
 
-To use the Matter protocol, enable the :kconfig:option:`CONFIG_CHIP` Kconfig option.
-Setting this option enables the Matter protocol stack and other associated Kconfig options, including :kconfig:option:`CONFIG_CHIP_ENABLE_DNSSD_SRP` that is required for the discovery of the Matter device using DNS-SD.
+To use the Matter protocol, set the following Kconfig options:
 
-After that, make sure to set the :kconfig:option:`CONFIG_CHIP_PROJECT_CONFIG` Kconfig option and define the path to the configuration file that specifies Vendor ID, Product ID, and other project-specific Matter settings.
+* :kconfig:option:`CONFIG_CHIP` - This option enables the Matter protocol stack and other associated Kconfig options, including :kconfig:option:`CONFIG_CHIP_ENABLE_DNSSD_SRP` that is required for the discovery of the Matter device using DNS-SD.
+* :kconfig:option:`CONFIG_CHIP_PROJECT_CONFIG` - This option defines the path to the configuration file that specifies Vendor ID, Product ID, and other project-specific Matter settings.
```

Comment:
After this line I would add information about dependencies between Matter and the other modules. Matter is only application layer protocol, so it utilizes many other software modules that needs to be configured, like Thread (or WiFi in the future), Bluetooth LE, mcuboot, mbedTLS, etc ... Maybe we shouldn't go into details, but just let user know that there are some other software modules than only Matter that may require configuration changes.